packages/react-reconciler/src/__tests__/ReactIncrementalErrorHandling-test.js

```diff
@@ -751,7 +751,9 @@ describe('ReactIncrementalErrorHandling', () => {
           (__DEV__
             ? " You likely forgot to export your component from the file it's " +
               'defined in, or you might have mixed up default and named imports.' +
-              '\n\nCheck the render method of `BrokenRender`.'
+              '\n\nCheck the render method of `BrokenRender`.' +
```

Comment:
Can we remove this line now? Seems unnecessary with the stack.

Reply:
Yes. I need to make the following two lines globs instead of hardcoded values too.